File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/test/structure/TestModel2.java
package com.raizlabs.android.dbflow.test.structure;

import com.raizlabs.android.dbflow.annotation.Column;
import com.raizlabs.android.dbflow.annotation.Table;
import com.raizlabs.android.dbflow.test.TestDatabase;

@Table(database = TestDatabase.class)
public class TestModel2 extends TestModel1 {
    @Column(name = "model_order")
    int order;

    public void setOrder(int order) {
        this.order = order;
    }
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/test/structure/TestModel2.kt
package com.raizlabs.android.dbflow.test.structure

import com.raizlabs.android.dbflow.annotation.Column
import com.raizlabs.android.dbflow.annotation.Table
import com.raizlabs.android.dbflow.test.TestDatabase

@Table(database = TestDatabase::class)
class TestModel2 : TestModel1() {

    @Column(name = "model_order")
    var order: Int = 0
}
